mvcc/kvstore_compaction.go

```diff
@@ -30,25 +30,74 @@ func (s *store) scheduleCompaction(compactMainRev int64, keep map[revision]struc
 	end := make([]byte, 8)
 	binary.BigEndian.PutUint64(end, uint64(compactMainRev+1))
 
-	batchsize := int64(10000)
+	batchsize := int64(1000)
 	last := make([]byte, 8+1+8)
+
+	compactKeys := make([][]byte, batchsize)
```

Comment:
`compactKeys := make([][]byte, 0, batchsize)`, since below it updates `compactKeys` with `append`?

Reply:
Good eye. Fixed.